### PR TITLE
Update definitions of enzyme to version 2.3.0.

### PR DESCRIPTION
**Pull request by MarianPalkus**
Definitions for enzyme (https://github.com/airbnb/enzyme) version 2.3.0.

### CODE REVIEW

**Comment by MarianPalkus** [comment]
:+1:

File: enzyme/enzyme.d.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// Type definitions for Enzyme v1.2.0
-// Project: https://github.com/airbnb/enzyme
+// Type definitions for Enzyme v2.3.0
+// Project: https://github.com/irbnb/enzyme
```
Comment (horiuchi) [inline enzyme/enzyme.d.ts]: Is that misstake?

Comment (MarianPalkus) [inline enzyme/enzyme.d.ts]: Yes, thanks - fixed the project url.

**Comment by horiuchi** [comment]
@MarianPalkus Thanks!

**Pull request merged**